Tools/Scripts/delete-if-too-large

```diff
@@ -41,14 +41,18 @@ def main():
             print(f'{directory} is not a directory')
             return 1
 
-        tenGigabytes = 1024 * 1024 * 1024 * 10
-        directorySize = sum(file.stat().st_size for file in directoryPath.rglob('*'))
-        if directorySize >= tenGigabytes:
-            os.remove(directory)
+        tenGigabytesInKiloBytes = 1024 * 1024  * 10
+        
+        #directorySize will be in kilobytes
```

Comment:
Please update comment style - there should be a space after "#", and it should be a full sentence with a period at the end.